include/suit_platform.h

```diff
@@ -10,6 +10,10 @@
 #include "suit_types.h"
 #include <stdint.h>
 
+#ifdef SUIT_PLATFORM_LEGACY_API_SUPPORT
```

Comment:
Shouldn't that be "CONFIG_SUIT_PLATFORM_LEGACY_API_SUPPORT"?

Reply:
It depends. I chose to go the "ZCBOR" way, so to use regular defines and map Kconfigs to them inside CMake.